client/hudwidget.cpp

```diff
@@ -1792,15 +1794,16 @@ void hud_unit_combat::paintEvent(QPaintEvent *event)
   QFont f = fcFont::instance()->getFont(fonts::default_font);
   QString ahploss, dhploss;
 
+  // TRANS: HP is Hip Point
   if (att_hp_loss > 0) {
-    ahploss = "-" + QString::number(att_hp_loss);
+    ahploss = "-" + QString::number(att_hp_loss) + " HP";
```

Comment:
It's always better to translate strings in full, in case the order must change (consider currencies, $10 is 10€). The underscore function is the translation:
```suggestion
    ahploss = QString(_("-%1 HP")).arg(att_hp_loss);
```
Same below...